salt/cloud/clouds/vultrpy.py

```diff
@@ -18,6 +18,18 @@
   # Vultr account api key
   api_key: <supersecretapi_key>
   driver: vultr
+  
+  
+Set up the cloud profile at ``/etc/salt/cloud.profiles`` or
+``/etc/salt/cloud.profiles.d/vultr.conf``:
+
+.. code-block:: yaml
+nyc-4gb-4cpu-ubuntu-14-04:
```

Comment:
To get this block to render properly in the docs, this entire example needs to be bumped in:
```
.. code-block:: yaml

    nyc-4gb-4cpu-ubuntu-14-04:
      location: 1
      provider: my-vultr-config
      image: 160
      size: 95
      enable_private_network: True
```